docs/semgrep-appsec-platform/jira.md

```diff
@@ -124,31 +148,41 @@ You can create tickets for Supply Chain findings using the **Triage** button on
 
 To create tickets:
 
+<!-- vale off-->
 1. If you're on the [**Supply Chain > Vulnerabilities**](https://semgrep.dev/orgs/-/supply-chain/vulnerabilities) page, select the findings for which you want tickets created. Otherwise, proceed to step 2.
-2. Click **Triage**.
-3. Set the status to **Fixing**, and select the **Create <span className="placeholder">NUMBER_OF_TICKETS</span> JIRA tickets in <span className="placeholder">PROJECT_NAME</span>** box that appears.
-4. Click **Change status** to proceed.
+1. Click **Triage**.
+1. Set the status to **Fixing** or **Reviewing**. Select **Fixing** if it is a known issue that needs to be fixed or **Reviewing** if the finding needs more investigation.
+1. Select the **Create tickets...** checkbox.
+    1. Click the first drop-down list to choose between making a ticket for **groups of findings** or an individual ticket for **each finding**.
+    1. Click the **JIRA project** drop-down list to select which Jira project to add the findings to.
+1. Optional: You can add **Comments** in the text box if you selected the **Reviewing** triage state.
+1. Click **Change status** to proceed.
```

Comment:
Is it intended that this is repeating the same thing above? I have the same comments as above here.

Reply:
Yup, updated both. Resolving.